src/components/dateTimePicker.js

```diff
@@ -41,6 +44,52 @@
     const [selectedDate, setSelectedDate] = useState(null);
     const helper = useText(helperText);
     const placeholderText = useText(placeholder);
+    const parsedMinDate =
+      // eslint-disable-next-line no-nested-ternary
+      useMinMaxDate === 'date'
+        ? new Date(useText(options.minDate))
+        : useMinMaxDate === 'days'
+        ? useText(minDaysBefore)
+        : undefined;
+    const parsedMaxDate =
+      // eslint-disable-next-line no-nested-ternary
+      useMinMaxDate === 'date'
+        ? new Date(useText(options.maxDate))
+        : useMinMaxDate === 'days'
+        ? useText(maxDaysAhead)
+        : undefined;
+
+    const getMinDate = value => {
+      if (!value) return undefined;
```

Comment:
When you extend this condition with the fact that `useMinMaxDate` might not be defined or has `none` as value, you can have even simpler conditions.

Something like this: 
```
const hasDateMinMax = useMinMaxDate && useMinMaxDate !== 'none';

const getMinDate = value => {
  if (!value || !hasDateMinMax) return undefined;
  if (useMinMaxDate === 'date') return value;
  const newMinDate = new Date();
  return newMinDate.setDate(newMinDate.getDate() - parseInt(value, 10));
};
```
And of course for the `getMaxDate` as well.